src/Grpc/HttpApi/test/Microsoft.AspNetCore.Grpc.HttpApi.IntegrationTests/Microsoft.AspNetCore.Grpc.HttpApi.IntegrationTests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
+
+    <RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);$(ArtifactsShippingPackagesDir)</RestoreAdditionalProjectSources>
```

Comment:
First line should match our default for test/ projects, second seems strange because we don't normally enable nullability in test/ projects (@pranavkm❔), and `$(RestoreAdditionalProjectSources)` doesn't do anything when you want to use the .dotnet/ layouts.
```suggestion
```

Same for a number of other projects